x/poe/contract/tgrade_oc_proposals_integration_test.go

```diff
@@ -14,6 +14,7 @@ import (
 )
 
 func TestSlashValidator(t *testing.T) {
+	t.Skip()
```

Comment:
👍  As discussed either I take care of this or we can have a tracking issue to not forget it.